action.yml

```diff
@@ -16,6 +16,25 @@ inputs:
     description: 'Limit the number of first x lines from commit messages that will be added in the issue description. No truncation when set to 0.'
     required: false
     default: 1
+# runs:
+#   using: 'node16'
+#   main: 'dist/index.js'
+
 runs:
-  using: 'node16'
-  main: 'dist/index.js'
+  using: 'composite'
+  steps:
+    - name: Checkout the fresh project
+      uses: actions/checkout@v3
+
+    - name: Fetch the tags
+      run: |
+        git fetch
+        git fetch --tags
+      shell: bash
+
+    - name: Do the fun stuff
+      uses: actions/github-script@v6
+      with:
+        notify-after: ${{inputs.notify-after}}
```

Comment:
this would probably generate an error because you are configuring it as "required: false" above (line 10) and it doesn't have a default value.